api-docs/rst/dev-guide/concepts.rst

```diff
@@ -13,15 +13,18 @@ How Autoscale works
 ~~~~~~~~~~~~~~~~~~~~~
 
 Rackspace Autoscale is an API-based tool that automatically scales
-resources in response to an increase or decrease in overall workload
-based on user-defined thresholds.
+resources ("scaling units", which can be cloud servers or cloud
+orchestration stacks) in response to an increase or decrease in overall
+workload based on user-defined thresholds.
 
 Autoscale calls the Rackspace Cloud Servers,
-Rackspace Cloud Load Balancers, and Rackspace RackConnect v3 APIs. All
-Rackspace Cloud Server create server configuration parameters can be
-used with Autoscale. For more information, see the following documentation:
+Rackspace Cloud Load Balancers, Rackspace RackConnect v3, and Rackspace
+Cloud Orchestration APIs. All Rackspace Cloud Server create server
+configuration parameters can be used with Autoscale. For more
+information, see the following documentation:
```

Comment:
You can use all the Rackspace Cloud Server create server configuration parameters to define Autoscale thresholds and policies for scaling operations. 

Revision note:  Not sure if this revision is technically accurate. The problem I'm trying to solve is .... what am I using these configuration parameters for.  Reading this paragraph, its not clear.  I'd have to go and review the concepts and examples to figure it out.   Look for a way to clarify if my revision is not correct.